modules/core/src/persistence.cpp

```diff
@@ -243,6 +256,13 @@ typedef struct CvFileStorage
     std::deque<char>* outbuf;
 
     base64::Base64Writer * base64_writer;
+    bool is_default_using_base64;
+    base64::fs::State state_of_writing_base64;  /**< used in WriteRawData only */
+
+    bool is_write_struct_delayed;
+    char* delayed_struct_key;
```

Comment:
@vpisarev, But `struct CvFileStorage` is a POD. Using The STL string may change it and cause some trouble. e.g. I remember `CvFileStorage` will be initialized via `memset`. But STL string cannot be initialized like this.